syft/frameworks/crypten/__init__.py

```diff
@@ -3,10 +3,44 @@
 import crypten.communicator as comm
 import crypten
 
+from syft.workers.base import BaseWorker
+
+
+RANK_TO_WORKER_ID = {}
```

Comment:
Add a comment to say that its a dict whose key are the CID and values are another dict which maps ranks to worker ids.

Also, the current way it's setup I think computation are synchronous and because you delete the key at the end of each computation, is there a scenario where there would be > 1 keys?

Reply:
A GridNode for example can receive different request to run a CrypTen computation at the same time, so the number of keys depends on the number of parallel requests.

Reply:
it doesn't affect the current way we are using it because we always run a single computation at a time, but I expect a production server to have this use case.